test1.html

```diff
@@ -39,15 +39,15 @@ <h1>SPECIALIZED CLINIC</h1>
   </header>
   <main class="page-main">
     <section class="breadcrumbs">
-      <a href="#"><img src="#banner"></a>
-      <ul>
-        <li>Home/</li>
-        <li>Health Service</li>
+      <a href="#"><img src="#banner" alt="banner"></a>
+      <ul class="breadcrumb-list">
+        <li class="breadcrumb-item">Home/</li>
+        <li class="breadcrumb-item">Health Service</li>
       </ul>
     </section>
     <aside class="sidebar-left">
       <article class="category">
```

Comment:
Should not be `article` in this case, because of this is the categories.
Recommend using `ul` `li`

Reply:
dạ e dùng div thay thế được phải k anh

Reply:
Recommend using `ul` `li`